packages/toolpad-app/src/server/auth.ts

```diff
@@ -98,18 +163,45 @@ export function createAuthHandler(project: ToolpadProject): Router {
     tenantId: process.env.TOOLPAD_AZURE_AD_TENANT_ID,
   });
 
+  const credentialsProvider = CredentialsProvider({
```

Comment:
I can move the mock results somewhere else maybe, if it feels weird to have them here like this.